src/__tests__/keyboard/getNextKeyDef.ts

```diff
@@ -0,0 +1,138 @@
+import {getNextKeyDef} from 'keyboard/getNextKeyDef'
```

Comment:
Just an idea, not necessarily a recommendation. I think this could be a good place for `jest-in-case` as lots of these tests look a lot like each other.

Reply:
I implemented it. Not sure if I like the look of the test file. But it did get rid of some repetition. :)